Library/Homebrew/test/gpg_spec.rb

```diff
@@ -13,7 +13,12 @@
         shutup do
           subject.create_test_key(dir)
         end
-        expect(dir/".gnupg/secring.gpg").to exist
+
+        begin
+          expect(dir/".gnupg/pubring.kbx").to exist
+        rescue RSpec::Expectations::ExpectationNotMetError
```

Comment:
Probably not, since this is currently quite rough, but it does essentially the same thing functionally. Feedback from someone more versed in RSpec is probably badly needed.